### DISCUSSION TITLE
New keys

### DISCUSSION THREAD

**Discussion by rPr0x** [DanSheps/netbox-secretstore#94] [General]
Hello, I would like your help with the Netbox Secret Store. I have a problem with the public and private keys. The initial public key of the admin user was deleted and after that I can't make it work anymore. I had saved the original public and private keys and tried to put them again but it is not working. I had installed Netbox v3.2.7 and Secret Store 1.2.0 and I ended up updating it to see if it fixed it but it didn't.

**Comment by DanSheps** [comment]
I don't believe this is recoverable.  You will need to create a new admin Pub/Priv and re-enter all secrets.

**Comment by rPr0x** [comment]
Have you seen this?

<div class="card border-danger mt-5">
<h5 class="card-header">
<i class="mdi mdi-alert"></i> Server Error
</h5>
<div class="card-body">

<p>
There was a problem with your request. Please contact an administrator.
</p>

<hr />
<p>
The complete exception is provided below:
</p>
<pre class="block"><strong>&lt;class &#x27;UnicodeDecodeError&#x27;&gt;</strong><br />
&#x27;utf-8&#x27; codec can&#x27;t decode byte 0xca in position 0: invalid continuation byte

Python version: 3.9.2
NetBox version: 3.3.5</pre>
<p>
If further assistance is required, please post to the <a href="https://github.com/netbox-community/netbox/discussions">NetBox discussion forum</a> on GitHub.
</p>
<div class="text-end">
<a href="/" class="btn btn-primary">Home Page</a>
</div>
</div>
</div>
</div>
</div>
</div>
</body>

</html>
